kuka_kr5_support/urdf/kr5_arc_macro.ros2_control.xacro

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
```

Comment:
This can be shared across all KUKA robots. I propose to move this into `kuka_resources`